homeassistant/components/fibaro/__init__.py

```diff
@@ -86,22 +87,22 @@ def __init__(self, config: Mapping[str, Any]) -> None:
 
         # Whether to import devices from plugins
         self._import_plugins = config[CONF_IMPORT_PLUGINS]
-        self._room_map = None  # Mapping roomId to room object
-        self._device_map = None  # Mapping deviceId to device object
+        self._room_map: dict[int, RoomModel]  # Mapping roomId to room object
+        self._device_map: dict[int, DeviceModel]  # Mapping deviceId to device object
```

Comment:
I saw that you removed the assignment.  Was that intentional? All of the other dicts have an empty dict assigned.  I can't see it breaking anything though.

Reply:
That was intentional, to assign None it is required to define it as `dict[int, RoomModel] | None` which is a bit wired as in the init method always a dict is created. Another possibility would be to assign an empty dict instead, do you think that would be better?

Reply:
I think the current makes sense 👍